inst/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -46,25 +52,26 @@ requirements, validating and reviewing the metadata, and uploading the dataset.
 
 ## General Process Overview
 
-1. Contact the AMP-AD team to discuss the study and the expected data. Receive
+1. Contact the Data Liaison (juliane.schneider@sagebase.org) to discuss the study and the expected data. Receive
```

Comment:
@Aryllen @avanlinden I was trying to figure out how to handle this, and went super granular because I want to try and keep the communication channel as simple as possible. What about using "AD Data Liaison" and the AMP-AD team email. When I go on vacation, I give Mette and Abby and the new data curator a heads up that they need to respond while I'm gone?